### DISCUSSION TITLE
如何让目录、插图目录、表格目录等居中？

### DISCUSSION THREAD

**Discussion by tsiayigong** [stone-zeng/fduthesis#339] [Q&A] [answered]
请问如何让这里的目录等几个字居中显示呢？
![image](https://github.com/stone-zeng/fduthesis/assets/63224494/8fbf855b-2093-499c-bdca-d980b0a8e4b0)

**Comment by stone-zeng** [comment, marked as answer]
居中是模板的默认行为，看看你还做了什么别的修改？

**Reply by tsiayigong** [reply]
好像确实是，我把\centering给覆盖掉了
